controllers/discovery/searchUser.js

```diff
@@ -1,6 +1,6 @@
 const {QueryTypes} = require('sequelize');
-
-const {sequelize} = require('../../databases/models');
+const {sequelize, UserBlockedUser} = require('../../databases/models');
+const UsersFunction = require('../../databases/functions/users');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `UserBlockedUser` and `UsersFunction` is a good addition to the file. It allows for the retrieval of blocked users, which can then be excluded from the search results.
```diff
+ const {sequelize, UserBlockedUser} = require('../../databases/models');
+ const UsersFunction = require('../../databases/functions/users');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->